### PR TITLE
Improve spec speed

### PR DESCRIPTION
**Pull request by petems**
## Removes large fixture file, even for non 1.9.3  …
* The test is to make sure that json is used
* The size of the json is irrelevant really, just that the right library is used and the json is parsed.
* Speeds up test suite a gobsmacking amount with parallel tests (~10 mins -> 1 minute)

## Adds parallel tests rake task  …

Comparison on an 8 core Mac:

```
$ bundle exec rake parallel_spec
746 examples, 0 failures, 10 pendings

Took 93 seconds (1:33)
```

Vs Vanilla spec:

```
$ bundle exec rake spec
Finished in 3 minutes 4.8 seconds (files took 2.14 seconds to load)
746 examples, 0 failures, 10 pending
```

## Changes travis to use parallel specs
Lets speed up that CI!

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -44,6 +46,13 @@ namespace :travis do
   end
 end
 
+desc "Parallel spec tests"
+task :parallel_spec do
+  require 'parallel_tests/cli'
+  Rake::Task[:spec_prep].invoke
+  ParallelTests::CLI.new.run('--type test -t rspec spec/classes spec/defines spec/unit'.split)
```
Comment (petems) [inline Rakefile]: Damn just realised I forgot functions, I did think the speed increase was a little _too_ impressive...

Comment (petems) [inline Rakefile]: nvm, there's only 2 specs in there, this thing really is fast! :100:

**Comment by petems** [comment]
Note: Repeated running on a dev machine make this faster: `ParallelTests::RSpec::RuntimeLogger` sorts tests into profiled chunks, decreasing running time :+1:

**Comment by petems** [comment]
Consistant 2 mins ~ specs in Travis :fireworks:

**Comment by rtyler** [comment]
Travis doesn't like you @petems 

```
Please use --help for a listing of valid options
invalid option: --random (defined in ENV['SPEC_OPTS'])
```

**Comment by petems** [comment]
Whoops, should be fixed now! :sweat:

**Comment by rtyler** [comment]
WHOOO

**Pull request merged**